Zend/zend_compile.c

```diff
@@ -9017,7 +9191,15 @@ zend_op *zend_compile_var(znode *result, zend_ast *ast, uint32_t type, int by_re
 			return NULL;
 	}
 }
-/* }}} */
+
+zend_op *zend_compile_var(znode *result, zend_ast *ast, uint32_t type, int by_ref, zend_bool allow_starting_short_circuiting) /* {{{ */
```

Comment:
The allow_starting_short_circuiting argument doesn't seem to be actually used.

Reply:
Oh yep that's dead code.